### ISSUE TITLE
[Feature Request] Add support for package denylist

### ISSUE THREAD

**Issue by frrad** [beartype/pytest-beartype#8]
Thank you so much for adding the `*` feature from https://github.com/beartype/pytest-beartype/issues/6 so quickly!!!

It works great! Perhaps predictably though, some of my packages don't pass. I'd like to be able to do "everything, except for x, y, z" so that I can fix the broken ones gradually while still opting in new packages automatically.

**Comment by leycec** [comment]
*...heh.* This intersects exactly with @beartype feature request beartype/beartype#407. Unfortunately, it looks like this is a tag-team effort, @tusharsadhwani. Specifically:

* On the @beartype side, I'll need to actually implement beartype/beartype#407 by supporting a new `BeartypeConf.claw_skip_package_names` configuration option. My eyes are watering with exhaustion. :face_holding_back_tears: 
* On the `pytest-beartype` side, I'll then beg you to expose a similar command-line configuration option (e.g., `--skip-package-names='awful_package, horrible_module'`).

Sadly, I'm the hold-up here. Multiple users are now beseeching the heavens for this. So, this will *definitely* happen. But... I'm just *sooo* slow. I promise to do this as soon as feasible. But... it's best not to trust my promises. Therefore, I also promise to immediately merge *any* PR implementing beartype/beartype#407.

If anyone's interested in a PR, let me know and I'd be just *delighted* to guide your capable keyboard fingers. :smile:

**Comment by leycec** [comment]
**Resolved by e6bf380.** Thanks a heap overflow to intrepid QA guru @tusharsadhwani for the recent resolution at PR #9! Problematic packages may now be ignored by passing a **package blacklist** (ban list, deny list) as the new `--beartype-skip-packages` option: e.g.,

```bash
--beartype-skip-packages='bad_package,good_package.awful_submodule'
```

![](https://media1.tenor.com/m/k026UfLCdoAAAAAd/garfield-garfield-muscle.gif)
<sup>*`pytest-beartype 0.2.0` flexes for GitHub*</sup>
